src/util/color.ts

```diff
@@ -113,11 +113,9 @@ export function getTitleAttr(bucket: IBucket, e: IEvent) {
   if (bucket.type == 'currentwindow') {
     return e.data.app;
   } else if (bucket.type == 'web.tab.current') {
-    try {
-      return new URL(e.data.url).hostname.replace('www.', '');
-    } catch (err) {
-      return e.data.url;
-    }
+    const domainRegex = /^.+:\/\/(?:www.)?([^/]+)/;
+    const match = e.data.url.match(domainRegex);
+    return match ? match[1] : e.data.url;
```

Comment:
Yes, I did measure. This is a huge offender when you needs to process tens of thousands of entries, and you can't really get away with memoization etc. since the URLs are likely unique (while the domains are relatively common).

Reply:
As for regexes, I typically just throw them into regex101.com. This should be well in the scope of a "maintainable" regex.